docs/workshops/10_multi_node_consolidation.md

```diff
@@ -0,0 +1,73 @@
+
+## Deploy NodePool:
+
+Use the following command to deploy a `NodePool`, and `AKSNodeClass` for Multi Node Consolidation, where we've enabled consolidation for when nodes are empty or underutilized, immediately after `0s`.
+
+```bash
+cd ~/environment/karpenter
+cat > multinode.yaml << EOF
+# This example NodePool will provision general purpose instances
+---
+apiVersion: karpenter.sh/v1
+kind: NodePool
+metadata:
+    name: default
+    annotations:
+        kubernetes.io/description: "Basic NodePool for generic workloads"
+spec:
+    disruption:
+        consolidationPolicy: WhenEmptyOrUnderutilized
+        consolidateAfter: 0s
+    limits:
+        cpu: "100"
+    template:
+        metadata:
+            labels:
+                # required for Karpenter to predict overhead from cilium DaemonSet
+                kubernetes.azure.com/ebpf-dataplane: cilium
+                eks-immersion-team: my-team
```

Comment:
`eks` - here and in a several other places

Reply:
Yea, so I changed this to an aks label. However, since we're telling them to re-use some scripts from aws, I found it rather cumbersome to have them have to modify it each time, so just aligned on the label. I'm going to ask the eks team if we could make this a generic label in their workshop, but I'm anticipating they might not want to do that so late minute. 

Post kubecon as I round out this docs with our own complete set of steps, I'll remove the eks mentions, and replace with a cloud agnostic tag.